_posts/2018-06-13-hackillinois-2019-mission.markdown

```diff
@@ -0,0 +1,38 @@
+
+HackIllinois 2018 also saw many engaging workshops and lightning talks, teaching everything from Unity development to making memes. Our sponsors and students always do a great job in offering supplemental knowledge and tools that you may not otherwise have had the opportunity to experience. Our sponsors are some of the biggest, best, and most cutting-edge players in the tech industry, and the value they provide to the event and the attendees cannot be overstated. Being able to take a break from your project is something all of our attendees should take advantage of, and we'll continue to develop these aspects of the event for the years to come.
+
+# Something New
+
+There are only 8 months, 2 days, and 4 hours until check-in, but who's counting? HackIllinois 2019 will be the sixth time Illinois students organize what has become the premier open source event available to undergraduates. While writing this statement, we wanted to explore and emphasize the varied roles that HackIllinois takes on. Our mission this year:
+
+>**To create an environment conducive to education, collaboration, and mentorship within the Open Source Community**
+
+We _create an enviroment_  in that our event is not defined by a simple "code for a prize" model. We plan to pack the entire event with content for attendees, from Illinois and over a dozen other universities, so that they can have a unique experience throughout the weekend. HackIllinois carries development, social, and networking opportunities, and these establish the three pillars -- _education, collaboration, and mentorship_ -- of our event. The theme this year follows the trail that has been blazed by our past directors and staff, but we wanted to assert our post _within the Open Source community_. We fully believe that our event betters the Open Source Community: by bringing together new students, continuing development on projects we believe in, and showcasing the talent pool that we have to offer.
+
+More specifically, thie year we want to make HackIllinois even more _exciting_. We are putting as much emphasis as we can on making this event more than 36 hours of coding; simply learning about open source is not enough at HackIllinois 2019. It will be the immersive experience of teamwork, guidance, and excitement that hackers have come to expect out of our staff. We will run interactive workshops, present lively mini-events, and host top industry talent for HackIllinois 2019. The Open Source experience will be a cut above that of the past: exciting to both the OSS gurus and those who have not yet explored this region of the technology world.
+
+# Something Borrowed
```

Comment:
I like the premise of this section but I think we can incorporate more of what we are already borrowing. I think a paragraph discussing ideas we've taken from other hackathons and events like R | P would make this section more rounded over the current plea for ideas.